src/composables/useStore.ts

```diff
@@ -11,6 +11,19 @@ const store = reactive({
   }
 });
 
+const notifications = ref<
+  {
+    id: string;
+    event: string;
+    time: number;
+    title: string;
+    spaceId?: string;
+  }[]
+>([]);
+
 export function useStore() {
-  return { store };
+  watch(notifications, () =>
+    notifications.value?.sort((a, b) => b.time - a.time)
+  );
+  return { store, notifications };
```

Comment:
I can't really make sense of this. Shouldn't notifications be part of store? Or... actually... why do we have something like a "store" in the first place? All global state is store, I'd say. We just don't call it like that usually. We could use Vuex to have a commen and battle-tested interface for global state updates and so on.

Reply:
This is our store, just needs to be build out more. https://vueschool.io/articles/vuejs-tutorials/state-management-with-composition-api/